Server/controllers/Bill/getLastBill.js

```diff
@@ -0,0 +1,22 @@
+const {
+    successMessage,
+    internalErrorMessage,
+  } = require('../../helpers/responseMessage');
+
+  const {getLastBill} = require('../../database/query/Bills')
+
+  module.exports = (req, res) => { 
+      const {bill_type } = req.params ;
+      const {limit} = req.body;
+      getLastBill(bill_type, limit)
```

Comment:
remove the limit from the query, and you need userId to return bills to this user.